site/pages/dashboard.page.tsx

```diff
@@ -256,11 +254,9 @@ const Dashboard = ({
             <Link className="govuk-link" href="/view-all-disruptions?draft=true">
                 <h2 className="govuk-heading-s text-govBlue">Draft disruptions</h2>
             </Link>
-            {stage !== "prod" && stage !== "preprod" && (
-                <Link className="govuk-link" href="/view-all-templates">
-                    <h2 className="govuk-heading-s text-govBlue">Templates</h2>
-                </Link>
-            )}
+            <Link className="govuk-link" href="/view-all-templates">
```

Comment:
Should we remove the feature flag after refactoring the code as per existing tasks in the to do column of Jira board?

Reply:
@lj-kpmg

Reply:
Refactoring won't impact functionality as discussed